linera-core/src/chain_worker/state/temporary_changes.rs

```diff
@@ -180,8 +180,9 @@ where
             .system
             .current_committee()
             .expect("chain is active");
-        let policy = committee.policy().clone();
         check_block_epoch(epoch, block)?;
+        let policy = committee.policy().clone();
```

Comment:
Clone is not necessary if you're passing by reference?

Reply:
We do a immutable borrow of `self.0.chain` on L178, but we also try to do a mutable borrow of it on L211. If we don't clone here we have to clone the whole chain state view on L211 or something like that